freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -33,6 +34,9 @@ class HandleBotasAdvisory(ContainerBuildHandler):
     """
     name = "HandleBotasAdvisory"
 
+    def __init__(self, pyxis=None):
+        self._pyxis = pyxis if pyxis else Pyxis(conf.pyxis_server_url)
```

Comment:
What if `conf.pyxis_server_url` is `''` (the default)?

Reply:
Well we have the same issue with Lightblue, so I think It will just have an error, the same error if url is invalid/unavailable when querying Pyxis.

Reply:
I think you could just set `def __init__(self, pyxis=None):` to `def __init__(self, pyxis=conf.pyxis_server_url):` instead. This should be fine since `conf.pyxis_server_url` is a string and thus a primitive variable type.

Reply:
But in constructor I wanted to have Pyxis instance not just url, so when you create that handler you can give him already existing Pyxis instance instead of creating new one.

Reply:
@apaplaus you're right. I misread the code.